ingest/lambdas/metadata-downloader.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from 'zod';
+
+import * as dspaceClient from '../../lib/dspace-client';
+import { getStringFromEnv } from '../../lib/env-utils';
+import { putJson, S3ObjectLocation } from '../../lib/s3-utils';
+
+const snsEventSchema = z.object({
+  Records: z.array(
+    z.object({
+      Sns: z.object({
+        Message: z.string().uuid(),
+      }),
+    })
+  ).nonempty(),
+});
+
+export const handler = async (event: unknown) => {
+  const snsEvent = snsEventSchema.parse(event);
+  const uuid = snsEvent.Records[0].Sns.Message;
+
+  const dspaceEndpoint = getStringFromEnv('DSPACE_ENDPOINT');
+  const dspaceItemBucket = getStringFromEnv('DOCUMENT_METADATA_BUCKET');
+
+  try {
+    const dspaceItem = await dspaceClient.getItem(
```

Comment:
We should be validating this because it's any right now but until we can sort out zod's passthrough() we're stuck trusting the response.

Reply:
Fixed in dspace-client and the response from `getItem` is being parsed.